joomla-projects/privacy-framework
# [Logging] Improve action logs

Pull Request for Issue #56 and #58.

### Summary of Changes
This PR makes some small improve to the logging component to solve the issue #56 and #58.

1. Remove ordering columns in action logs list
2. Store ID of item in the database for easier searching for all actions related to an item if needed
3. Small improvement to search
- Allow searching base on username of the user perform action instead of the message (message is actually a json encoded data now, so searching on it doesn't make sensen
- Add support for id: prefix to search for action log record ID and item_id:id prefix to search for action logs related to an item with given ID

### Testing Instructions
Code review should be enough. Otherwise, you will need to download a copy of my branch, install, test and make sure action logs still being stored properly.
